topology/probes/k8s/k8s.go

```diff
@@ -115,12 +115,12 @@ func NewK8sProbe(g *graph.Graph) (*Probe, error) {
 		newIngressServiceLinker,
 		newNamespaceLinker,
 		newNetworkPolicyLinker,
-		newServicePodLinker,
+		// FIXME: newServicePodLinker,
```

Comment:
Why ?

Reply:
disabled intentionally - the linker has to be refined as a result the number of links will be reduced significantly.

Reply:
Is there a bug in the current implementation ?